### PR TITLE
refactor: migrate to native function nonEmptyStringAndNotWhitespace

### PR DESCRIPTION
**Pull request by ivankatliarchuk**
## Changes

-  Update sindresorhus/is
-  [Use native function](https://github.com/sindresorhus/is/blob/65ea91297ede74046b19a65c434411a39c16dacd/source/index.ts#L356)

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [x] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by ivankatliarchuk** [comment]
Not sure what is the process e.g. to bump a dependency

**Comment by HonkingGoose** [comment]
> Not sure what is the process e.g. to bump a dependency

Normally the maintainers request a update from our Dependency Dashboard and let Renovate bot update the dependency on `main`. Then you can make the changes on top of that.

**Comment by HonkingGoose** [comment]
By the way there's a typo in the pull request title.

It should be `function`. 😉

**Review by rarkins — APPROVED** [review]

File: lib/workers/repository/update/pr/index.spec.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
   platform,
 } from '../../../../../test/util';
 import { PlatformId } from '../../../../constants';
-import type { Pr } from '../../../../modules/platform/types';
+import type { Pr } from '../../../../modules/platform';
```
Comment (viceice) [inline lib/workers/repository/update/pr/index.spec.ts:9]: I think this was an unintended change. 😉

Comment (ivankatliarchuk) [reply lib/workers/repository/update/pr/index.spec.ts:9]: Yeah. Should not do this. Just `/types` not required there

**Pull request merged**